Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -158,7 +160,7 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public C
     void getFrameInfo(CompletionHandler<void(FrameTreeNodeData&&)>&&);
     FrameTreeCreationParameters frameTreeCreationParameters() const;
 
-    WebFrameProxy* parentFrame() { return m_parentFrame.get(); }
+    WebFrameProxy* parentFrame() const { return m_parentFrame.get(); }
```

Comment:
Can we make this return a RefPtr while we're here?

Reply:
I would need to change several call sites from `auto*` -> `RefPtr`. Seems like it should be a separate change?

Reply:
yes